docs/communication/event-emitter/event-emitter.md

```diff
@@ -120,3 +120,23 @@ For these reasons, we recommend you **always use kebab-case for event names.**
 ## Additional Links
 
 - Vue.js official docs - [Custom Events](https://vuejs.org/v2/guide/components-custom-events.html)
+
+## Project Tasks
+
+After taking a look at some interesting concepts about Event Emitting and Props, let's try to use these concepts in the tasks below:
+
+1. Let's integrate our track list component with our searchlightbox components, so that we can render our search results using our track list.
+2. Let's add a new component to view more details about each track (as mentioned in the initial scope).
+3. Using the emitt patter, let's give it a try and build the favorite feature within our home.
```

Comment:
There's a typo in:
3. Using the emitt **pattern**

Reply:
I think it's **tracklist** without spacing.

Reply:
_searchlightbox components_ is intended to use plural here ?

Reply:
_ emitt patter_ not sure if it is intended to use **emit** or **emitter** here.

Reply:
Solved, all:

1. Corrected emit patter
2. Fixed plural syntax
3. Fixed tracklist workd